packages/zoe/test/unitTests/zcf/setupZcfTest.js

```diff
@@ -48,7 +48,7 @@ export const setupZCFTest = async (issuerKeywordRecord, terms) => {
   ).startInstance(installation, issuerKeywordRecord, terms);
   const { vatAdminState } = fakeVatAdmin;
   // @ts-ignore fix types to understand that zcf is always defined
-  assert(zcf !== undefined);
+  assert(zcf !== undefined, 'zcf is required');
```

Comment:
We can help the tester a little more here. Also it's idiomatic to check for presence with a truthiness.

```suggestion
    // @ts-expect-error setZCF may not have been called yet
    assert(zcf, 'zcf is required; did you forget to setZCF?');
```

Reply:
done